### ISSUE TITLE
Where is pjax.js located(bower or ZIP download) ?

### ISSUE THREAD

**Issue by Madebym** [MoOx/pjax#57]
Ok, this may be that I am a total idiot, cause I am a designer first and foremost, but I can't find the `pjax.js` file.

I've tried both the ZIP download and the bower installation, but there is no such file.

**Comment by darylteo** [comment]
@Madebym 

Current stable should be available through NPM and Bower. 
Current master is available through npm at the moment by running until #55 is processed. 

````
npm install --save git+https://github.com/moOx/pjax.git
````

This project was quite stagnant for awhile so some things are out of date.

Short survey: do you use bower regularly? Or have you considered using browserify/webpack or similar?

**Comment by Madebym** [comment]
@darylteo 

Thanks for the quick reply. First I want to repeat that I am a designer by trade, so I am not that good with all of this front-end automation tools. I know just enough to get by :))

So, I tried the regular ZIP download, then the bower and just now your proposed  `npm install --save git+https://github.com/moOx/pjax.git`

The `package.json` and `bower.json` both say that the main file is located in `src/pjax.js` , but there is no directory called src.

Again, this may be that I really missed something obvious, but until now I haven't had any issues using bower or npm to intstall various libraries.

p.s. Yes, I use bower regularly. I've heard about browserify/webpack, but haven't used it yet. Bower is fine for my needs.

**Comment by darylteo** [comment]
@Madebym ah okay I see the problem. At version 0.1.4 there WAS a standalone pjax.js. As of 0.2.* this was changed as Maxime split all the files up into components. As it stands, there isn't a standalone pjax.js... it needs to be compiled by a bundler like webpack or browserify. 

I was just discussing whether we should be including a standalone distributed artifact. Your input would be appreciated (you can see that discussion in #55). In the meantime, @MoOx should be done with that by the end of week.

**Comment by Madebym** [comment]
@darylteo 

Thanks for clearing things for me. I already managed to find that 0.1.4 version which had the standalone pjax.js file, should be enough for me to play a bit with pjax.

As for the standalone distributed file, my vote is that you include it. Why set a prerequisite on being able to use this library? I bet there will be people like me that don't use any browserify/webpack/you-name-it things, but would like to use the script.

For me it is overcomplicating things where it is absolutely not needed. At least have the option of standalone file.

I just now read one interesing article, could be applied to this situation, although they are not that closely tied. 
https://medium.com/@wob/the-sad-state-of-web-development-1603a861d29f#.grqx90qt1

Bottom line, just give me the file, I want to build things :))

**Comment by MoOx** [comment]
Reopening. Future version won't have a versionned standalone JS. Instead we will provide a link to https://npmcdn.com

**Comment by commercial-hippie** [comment]
A standalone version would be nice for people who use their own CDN.

Is there any chance someone has an example of how to concat the files using gulp?  I have tried using [`gulp-amd-optimizer`](https://www.npmjs.com/package/gulp-amd-optimizer) and `concat` but its not working out. :disappointed:

**Comment by darylteo** [comment]
@commercial-hippie either webpack or browserify will work. Just point it at pjax/index.js

**Comment by MoOx** [comment]
Will be in npm and bower for now (bower main will point to this file, npm will not due to its nature)

**Comment by BlackGlory** [comment]
Obviously, Bower still not have pjax.js file, https://npmcdn.com/pjax@0.2.4 same.

**Comment by leifniem** [comment]
this seems to remain an issue for now

**Comment by badoubadou** [comment]
Would love to have a standalone file too.

**Comment by david-treblig** [comment]
I'm voting to reopen as well. All building scripts seem to fail me right now and auto injection of assets fail because build version doesn't exist or can't be produced...

**Comment by acrolink** [comment]
Would any one tell us how to build that holy JS file, just include it and that is it!

**Comment by MoOx** [comment]
https://unpkg.com/pjax/pjax.js
